src/main/java/net/dv8tion/jda/internal/entities/automod/TriggerMetadataImpl.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.internal.entities.automod;
+
+import net.dv8tion.jda.api.entities.automod.KeywordPresetType;
+import net.dv8tion.jda.api.entities.automod.TriggerMetadata;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Objects;
+
+public class TriggerMetadataImpl implements TriggerMetadata
+{
+
+    private List<String> keywords;
+    private List<KeywordPresetType> keywordPresets;
+
+    @Override
+    @Nonnull
+    public List<String> getKeywords()
+    {
+        return keywords;
```

Comment:
99.9% of the times you return a collection it should be unmodifiable, so please wrap this with Collections.unmodifiableList, same for all the other methods